bitski-provider/src/rest_provider.rs

```diff
@@ -47,7 +47,7 @@ impl RestProvider {
         #[cfg(feature = "tracing")]
         tracing::debug!(
             "[id:{}] sending request: {:?} to {}",
-            id,
+            "id", // TODO fix
```

Comment:
Is this `client_id`?

Reply:
Not sure. @ptescher? When running `cargo clippy --all-features --all-targets --workspace` it would throw an error

Reply:
oh, this should be request.id, its the ID field from jsonrpc.

Reply:
Fixed